packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -4422,6 +4447,20 @@ class SemanticsConfiguration {
     _hasBeenAnnotated = true;
   }
 
+  /// A function that decide how configuration should merge with each other.
+  ///
```

Comment:
Maybe document somewhere (either here or on the SemanticsMergeRule class or probably on both) what exactly the merger gets to decide: e.g.

The merger function gets the SemanticsConfigurations of the direct descendent nodes as input and can decide which of these should be merged together to form child SemanticsNodes and which of them should be merged upwards into the parent SemanticsNode.